sycl/doc/extensions/experimental/sycl_ext_oneapi_annotated_ptr.asciidoc

```diff
@@ -526,12 +526,12 @@ class annotated_ref {
     operator T() const;
     T operator=(const T &) const;
     T operator=(const annotated_ref&) const;
-    // OP is: +=, -=, *=, /=, %=, <<=, >>=, &=, |=, ^=
-    T operatorOP(const T &) const;
-    T operator++();
-    T operator++(int);
-    T operator--();
-    T operator--(int);
+    // OP is: +, -, *, /, %, <<, >>, &, |, ^
```

Comment:
I think it reads easier if = is part of the OP.

Reply:
done